docker/build.sh

```diff
@@ -1,12 +1,13 @@
 #!/bin/bash -xe
 #########Set Me###############
-export PYVER=${PYVER:-2.7}
+export PYVER=${PYVER:-3.5}
```

Comment:
Not needed.

Reply:
Why should we keep old version as default?

Reply:
Sounds fair